state/etcd/etcd.go

```diff
@@ -53,6 +54,8 @@ type etcdConfig struct {
 	CA        string `json:"ca"`
 	Cert      string `json:"cert"`
 	Key       string `json:"key"`
+	// Transaction server options
```

Comment:
Nit:

```suggestion
	// Transactional store options
```

Reply:
Not what I was going for. I want to be clear that these options must match options configured on the server. Other suggestions @ItalyPaleAle ?

Reply:
It was just a nit.